VocaDbWeb/Controllers/Api/UserApiController.cs

```diff
@@ -814,5 +814,13 @@ public class PostStatusLimitedModel
 		[Route("{id:int}/status-limited")]
 		[ApiExplorerSettings(IgnoreApi = true)]
 		public void PostStatusLimited(int id, [FromBody] PostStatusLimitedModel model) => _queries.SetUserToLimited(id, model.Reason, WebHelper.GetRealHost(Request), model.CreateReport);
+
+		[Route("{id:int}/followedArtists/{artistId:int}")]
+		public bool GetFollowedArtist(int id, int artistId) => _queries.GetFollowedArtist(id, artistId);
+
+		[Route("current/followedArtists/{artistId:int}")]
+		[Authorize]
+		[AuthenticatedCorsApi(System.Web.Mvc.HttpVerbs.Get)]
+		public bool GetFollowedArtist(int artistId) => GetFollowedArtist(_permissionContext.LoggedUserId, artistId);

```

Comment:
What should be the URL? Is it enough just to return a `bool`?

Reply:
The URL is fine I guess. I notice the API guidelines for VocaDB mention camelCase, although I think kebab-case is more common these days.